zipkin-server/http-query-plugin/src/main/resources/META-INF/services/org.apache.skywalking.oap.server.library.module.ModuleProvider

```diff
@@ -0,0 +1,19 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
The header is somehow wrong, please fix it. And consider skywalking-eyes to verify headers. Notice, I am not sure year range is supported or not.

Reply:
Got it, The eyes could [verify the header by regex](https://github.com/apache/skywalking-eyes/blob/main/.licenserc.yaml#L40-L57), Therefore, I believe it is capable of verifying the year range.